components/search_engines/brave_prepopulated_engines.cc

```diff
@@ -152,6 +153,16 @@ const PrepopulatedEngine duckduckgo_lite = {
 };
 #endif
 
+const PrepopulatedEngine brave_ecosia = 
+    ModifyEngineParams(ecosia,
+                       NULL,
+                       L":e",
+                       "https://www.ecosia.org/search?tt="
+                       "42b8ae98"
```

Comment:
could you start a wiki page that lists all the search refcodes that we use? makes it easier for support to tell users that the ID is non-unique

Reply:
Great idea! Will do 😄

Reply:
@diracdeltas created https://github.com/brave/brave-browser/wiki/Search-engine-integrations 👍